Game/ItemsFigures.pde

```diff
@@ -131,9 +131,12 @@ final String itemTypePacman = "Pacman";
 public class Pacman extends Figure {
   private int playerId; // Valid values: 1, 2
   private int score;
+  private int lives;
+  private Page page; // Add a Page field
 
-  public Pacman(int playerId, float w, float h) {
+  public Pacman(Page page, int playerId, float w, float h) {
```

Comment:
Now no need to add a `Page` parameter to the `Pacman` constructor. You can directly use the variable `page` in the code.

Reply:
Ah amazing thanks, so much easier that way. Made the change and committed again